apm-agent-plugins/apm-profiling-plugin/src/main/java/co/elastic/apm/agent/profiler/ProfilingFactory.java

```diff
@@ -58,6 +58,16 @@ public void start(ElasticApmTracer tracer) {
         tracer.registerSpanListener(new ProfilingActivationListener(tracer, profiler));
     }
 
+    @Override
+    public void pause() throws Exception {
+        // todo - implement
```

Comment:
why we have todos left here ? should we implement them or leave an empty impl. ?

Reply:
Yes, I left that to @felixbarny with a comment at the top, I hope it was not missed...